src/diffusers/loaders/__init__.py

```diff
@@ -64,6 +64,8 @@ def text_encoder_attn_modules(text_encoder):
         _import_structure["textual_inversion"] = ["TextualInversionLoaderMixin"]
         _import_structure["ip_adapter"] = ["IPAdapterMixin"]
 
+_import_structure["peft"] = ["PeftAdapterMixin"]
```

Comment:
This is out of the condition because:

* It's gonna be directly used in the core models (wherever applicable). 
* In our core test environment, we don't install `peft`. So, if we add the above under `is_peft_available()`, we will run into import errors. 

Can move it under `is_torch_available()`, though. 

@DN6 let me know if you have a better way of handling this.